README.md

```diff
@@ -46,243 +14,84 @@ After you're finished please remove all the comments and instructions!
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
```

Comment:
- [ ] Every section of the [README FILE](https://github.com/microverseinc/readme-template) provided is very important. please fo not commit them.

## A shown:
![image](https://user-images.githubusercontent.com/50990925/211817563-fd18e3e4-ea5c-4f78-b6c5-3e4ca82fe232.png)